usaspending_api/broker/helpers/last_delta_table_load_version.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+from usaspending_api.broker import lookups
+from usaspending_api.broker.models import DeltaTableLoadVersion
+
+logger = logging.getLogger("script")
+
+
+def get_last_delta_table_load_versions(key):
+    # TODO - Pydoc
+    delta_table_load_version_id = lookups.LOAD_VERSION_TYPE_DICT[key]
```

Comment:
I think it would be good to either:
* Use `lookups.LOAD_VERSION_TYPE_DICT.get(key)` to prevent a possible `KeyError`.
* Catch the possible `KeyError` if you want keep the current indexing code.

Reply:
I think throwing a KeyError here should be the intended behavior. Not really a way to gracefully recover from a key missing from the lookup.